looprpc/client.proto

```diff
@@ -43,6 +43,11 @@ service SwapClient {
     */
     rpc SwapInfo (SwapInfoRequest) returns (SwapStatus);
 
+    /* loop: `abandonswap`
```

Comment:
Commenting here because github hides it. The server.proto changes are missing from the next commit.

Reply:
😬 how did this happen